xtask/src/main.rs

```diff
@@ -6,32 +6,33 @@ use std::{
     process::{Command, Stdio},
 };
 
+fn cli() -> clap::Command {
+    clap::Command::new("xtasks")
```

Comment:
please use the struct form for command definitions, it‘s considerably easier to maintain and reason about

Reply:
for those of you who are well versed in rust compiler magic, sure. I personally find this form more readable.

That said, I'm sure I'll come to learn this compiler magic in due time, and will defer to you on all things Rust. Will change to the struct form

Reply:
I am looking forward to once your brain has embraced the strength and statics of types 💙